### PR TITLE
[#168192290] UI review: badge, list items

### PR DESCRIPTION
**Pull request by Undermaken**
This PRs reviews some UI elements with a specific focus on **_badge_** and **_list item_** (profile, messages-list, services-list, transactions-list).
It wants to improve the look and feel and make elements more homogeneous through the whole app.

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🌟 [#168192290](https://www.pivotaltracker.com/story/show/168192290): Review Grafica su items: lista messaggi, lista servizi, sezione profile

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: ts/components/messages/MessageAgenda.tsx [hunk 1]
```diff
@@ -78,8 +78,9 @@ const styles = StyleSheet.create({
   },
   sectionHeaderText: {
     fontSize: 18,
-    lineHeight: 20,
-    color: customVariables.brandDarkestGray
+    color: customVariables.brandDarkestGray,
+    ...makeFontStyleObject(Platform.select, "600", false),
```
Comment (valentinacala) [inline ts/components/messages/MessageAgenda.tsx]: `makeFontStyleObject` uses as default value `isItalic = false`
```suggestion
    ...makeFontStyleObject(Platform.select, "600"),
```

File: ts/components/messages/MessageListItem.tsx [hunk 2]
```diff
@@ -85,13 +88,14 @@ const styles = StyleSheet.create({
...
+    flexDirection: "row",
     flex: 1,
-    paddingRight: 32
+    paddingRight: 36
   },
 
   messageTitle: {
-    ...makeFontStyleObject(Platform.select, "700", false),
-    fontSize: 16,
+    ...makeFontStyleObject(Platform.select, "600", false),
```
Comment (valentinacala) [inline ts/components/messages/MessageListItem.tsx]:
```suggestion
    ...makeFontStyleObject(Platform.select, "600"),
```

File: ts/components/messages/MessageAgenda.tsx (hunk 1 above)
Comment (valentinacala) [inline ts/components/messages/MessageAgenda.tsx]: `makeFontStyleObject` has isItalic default value at false
```suggestion
    ...makeFontStyleObject(Platform.select, "600"),
```

File: ts/components/messages/MessageListItem.tsx (hunk 2 above)
Comment (valentinacala) [inline ts/components/messages/MessageListItem.tsx]:
```suggestion
    ...makeFontStyleObject(Platform.select, "600"),
```

File: ts/components/screens/ListItemComponent.tsx [hunk 3]
```diff
@@ -45,11 +46,14 @@ const styles = StyleSheet.create({
     flex: 1
   },
   serviceName: {
-    fontWeight: "700",
+    fontSize: 18,
+    color: customVariables.brandDarkestGray,
+    ...makeFontStyleObject(Platform.select, "600", false),
```
Comment (valentinacala) [inline ts/components/screens/ListItemComponent.tsx]:
```suggestion
    ...makeFontStyleObject(Platform.select, "600"),
```

File: ts/components/screens/SectionHeaderComponent.tsx [hunk 4]
```diff
@@ -30,7 +31,7 @@ const styles = StyleSheet.create({
     alignItems: "center"
   },
   sectionTitle: {
-    fontWeight: "400",
+    ...makeFontStyleObject(Platform.select, "700", false),
```
Comment (valentinacala) [inline ts/components/screens/SectionHeaderComponent.tsx]:
```suggestion
    ...makeFontStyleObject(Platform.select, "700"),
```

File: ts/components/services/ServiceListItem.tsx [hunk 5]
```diff
@@ -30,15 +34,11 @@ const styles = StyleSheet.create({
...
-  headerWrapper: {
-    flexDirection: "row",
-    marginBottom: 4,
-    marginLeft: 10
-  },
-  badgeMargin: {
-    marginLeft: 4
+    fontSize: 18,
+    color: variables.brandDarkestGray,
+    ...makeFontStyleObject(Platform.select, "600", false),
```
Comment (valentinacala) [inline ts/components/services/ServiceListItem.tsx]:
```suggestion
    ...makeFontStyleObject(Platform.select, "600"),
```

Comment (Undermaken) [reply ts/components/services/ServiceListItem.tsx]: Fixed within https://github.com/teamdigitale/io-app/pull/1280/commits/65755be70601628e243b982dde5b7e37f572e649

File: ts/components/screens/SectionHeaderComponent.tsx (hunk 4 above)
Comment (Undermaken) [reply ts/components/screens/SectionHeaderComponent.tsx]: Fixed within https://github.com/teamdigitale/io-app/pull/1280/commits/65755be70601628e243b982dde5b7e37f572e649

File: ts/components/screens/ListItemComponent.tsx (hunk 3 above)
Comment (Undermaken) [reply ts/components/screens/ListItemComponent.tsx]: Fixed within https://github.com/teamdigitale/io-app/pull/1280/commits/65755be70601628e243b982dde5b7e37f572e649

File: ts/components/messages/MessageAgenda.tsx (hunk 1 above)
Comment (Undermaken) [reply ts/components/messages/MessageAgenda.tsx]: Fixed within https://github.com/teamdigitale/io-app/pull/1280/commits/65755be70601628e243b982dde5b7e37f572e649

File: ts/components/messages/MessageListItem.tsx (hunk 2 above)
Comment (Undermaken) [reply ts/components/messages/MessageListItem.tsx]: Fixed within https://github.com/teamdigitale/io-app/pull/1280/commits/65755be70601628e243b982dde5b7e37f572e649

File: ts/components/messages/MessageAgenda.tsx (hunk 1 above)
Comment (Undermaken) [reply ts/components/messages/MessageAgenda.tsx]: Fixed within https://github.com/teamdigitale/io-app/pull/1280/commits/65755be70601628e243b982dde5b7e37f572e649

**Pull request merged**